packages/form-builder/src/components/sidebar/TabPanel.tsx

```diff
@@ -77,19 +77,15 @@ export default function TabPanel({
         <div className={className}>
             <NavigableMenu
                 role="tablist"
-                orientation={orientation}
+                orientation={orientation as 'vertical'}
```

Comment:
@jonwaldstein I've not seen this syntax before. From what I gather, it seems to be a type assertion, but the syntax seems backwards? I would think that `orientation` is a type (maybe an Enum), with a value of `vertical`, but this reads (to me) as `vertical` being the type definition of the variable `orientation`? Could you clarify?

Reply:
Here's another way to do it: c6e7d5b5